tests/provisioner-tests/test/provisioner/suite.go

```diff
@@ -47,19 +56,44 @@ func NewTestSuite(config testkit.TestConfig) (*TestSuite, error) {
 	time.Sleep(15 * time.Second)
 
 	provisionerClient := provisioner.NewProvisionerClient(config.InternalProvisionerURL, config.Tenant, config.QueryLogging)
+	directorClient, err := newDirectorClient(config)
+	if err != nil {
+		return nil, err
+	}
 
 	testId := randStringBytes(8)
 
 	return &TestSuite{
-		TestId:            testId,
+		TestId: testId,
+
 		ProvisionerClient: provisionerClient,
+		DirectorClient:    directorClient,
 
 		gardenerProviders: config.Gardener.Providers,
 
 		config: config,
 	}, nil
 }
 
+func newDirectorClient(config testkit.TestConfig) (director.Client, error) {
+	k8sConfig, err := restclient.InClusterConfig()
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to get in cluster config")
+	}
+	coreClientset, err := kubernetes.NewForConfig(k8sConfig)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to create core clientset")
+	}
+	secretClient := coreClientset.CoreV1().Secrets(config.DirectorClient.Namespace)
+	httpClient := &http.Client{Transport: &http.Transport{TLSClientConfig: &tls.Config{InsecureSkipVerify: true}}}
+	graphQLClient := gql.NewGraphQLClient(config.DirectorClient.URL, true, config.QueryLogging)
+	oauthClient := oauth.NewOauthClient(httpClient, secretClient, config.DirectorClient.OauthCredentialsSecretName)
+	if err := oauthClient.WaitForCredentials(); err != nil {
+		return nil, errors.Wrap(err, "timeout waiting for credentials")
+	}
```

Comment:
Do we need to wait for credentials? I think tests will be run after installation is finished and jobs creating credentials are part of the installation.

Reply:
Yup, it may be redundant

Reply:
Deleted!